.github/workflows/release.yml

```diff
@@ -1,251 +1,247 @@
+        steps:
+            # This is necessary for generating the changelog. It has to come before "Download Artifacts" or else it deletes the artifacts.
+            - name: Checkout sources
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            # ==============================
+            #       Download artifacts
+            # ==============================
+
+            - name: Download artifacts
+              uses: actions/download-artifact@v4
+
+            # ==============================
+            #       Create release draft
+            # ==============================
+
+            - name: Generate Full Changelog
+              id: changelog
+              run: |
+                echo "CHANGELOG<<EOF" >> $GITHUB_OUTPUT
+                echo "$(git log --pretty=format:"- %s" $(git describe --tags --abbrev=0 ${{ env.VERSION }}^)..${{ env.VERSION }})" >> $GITHUB_OUTPUT
+                echo "EOF" >> $GITHUB_OUTPUT
+
+            - name: Create Release Draft
+              env:
+                  GITHUB_USER: ${{ github.repository_owner }}
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+              # The formatting here is borrowed from OpenEthereum: https://github.com/openethereum/openethereum/blob/main/.github/workflows/build.yml
+              run: |
+                  body=$(cat <<- "ENDBODY"
+                  <Rick and Morty character>
+
+                  ## Testing Checklist (DELETE ME)
+
+                  - [ ] Run on synced Holesky Sigma Prime nodes.
+                  - [ ] Run on synced Canary (mainnet) Sigma Prime nodes.
+                  - [ ] Resync a Holesky node.
+                  - [ ] Resync a mainnet node.
+
+                  ## Release Checklist (DELETE ME)
+
+                  - [ ] Merge `unstable` -> `stable`.
+                  - [ ] Ensure docker images are published (check `latest` and the version tag).
+                  - [ ] Prepare Discord post.
+                  - [ ] Prepare Twitter post.
+                  - [ ] Prepare mailing list email.
+
+                  ## Summary
+
+                  Add a summary.
+
+                  ## Update Priority
+
+                  This table provides priorities for which classes of users should update particular components.
+
+                  |User Class |Beacon Node  | Validator Client|
+                  --- | --- | ---
+                  |Staking Users| <TODO> | <TODO> |
+                  |Non-Staking Users| <TODO>|---|
+
+                  *See [Update
+                  Priorities](https://anchor-book.sigmaprime.io/installation-priorities.html)
+                  more information about this table.*
+
+                  ## All Changes
+
+                  ${{ steps.changelog.outputs.CHANGELOG }}
+
+                  ## Binaries
+
+                  [See pre-built binaries documentation.](https://anchor-book.sigmaprime.io/installation-binaries.html)
```

Comment:
We will need to make sure this URL exists, i.e add it to the md-book. @jking-aus